docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1647 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    )
</pre><p>It would be nice if were able to make:</p><p> </p><ol><li>strings that have less baggage associated with them</li><li>Pick some distribution of lengths that is perhaps realistic. One could either have a normal distribution, or else perhaps a heavy-tailed log-normal. Here's some recent discussion on statistical models of sentence length: https://aclanthology.org/W19-5710.pdf</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>